.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.2
```

Comment:
Thanks, I hadn't realised that. We've used a [different approach](https://github.com/gds-operations/vcloud-launcher/pull/106) in other projects, though both seem valid.